client/src/components/Common/EditEvent/EditEvent.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from "react";
+import Calender from "../../../Images/calendar_icon.png";
+import Clock from "../../../Images/clock_icon.png";
+import RoomIcon from "../../../Images/room_icon.png";
+import "./editEvent.css";
+import axios from "axios";
+import { Link } from "react-router-dom";
+import Popup from "./Popup";
+class EditEvent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userId: 3,
+      date: "2020-05-23T11:00:00.000Z",
+      roomId: 5,
+      room_name: "Moscow",
+      showPopup: false,
+      event_id: "",
+        event_id: "",
```

Comment:
you have `event_id` twice here. Also, you are mixing snake_case with camelCase in the variable names, you should stick to just one (for JS variable names it should be camelCase)